ethereum/src/MessageQueue.sol

```diff
@@ -52,6 +52,11 @@ contract MessageQueue is IMessageQueue {
         VaraMessage calldata message,
         bytes32[] calldata proof
     ) public {
+        if (IRelayer(RELAYER_ADDRESS).emergencyStop()) {
+            // Emergency stop is active, stop processing messages.
+            return;
+        }
```

Comment:
I think it's better to revert inside `getMerkleRoot` to guarantee that `RelayerContract` is totally unuseful when emergency stop occured

Reply:
Fixed